OrbitGl/EventTrack.cpp

```diff
@@ -152,3 +181,37 @@ bool EventTrack::IsEmpty() const {
       GEventTracer.GetEventBuffer().GetCallstacks()[m_ThreadId];
   return callstacks.empty();
 }
+
+//-----------------------------------------------------------------------------
+std::string EventTrack::GetSampleTooltip(PickingID id) const {
+  auto SafeGetFormattedFunctionName = [](uint64_t addr) -> std::string {
+    auto it = Capture::GAddressToFunctionName.find(addr);
+    return it == Capture::GAddressToFunctionName.end()
+               ? std::string("<i>") + "???" + "</i>"
+               : it->second;
+  };
+
+  static const std::string unknown_return_text = "Function call information missing";
+
+  auto user_data = time_graph_->GetBatcher().GetUserData(id);
+  if (!user_data || !user_data->custom_data_) {
+    return unknown_return_text;
+  }
+
+  CallstackEvent* callstack_event =
+      static_cast<CallstackEvent*>(user_data->custom_data_);
+  auto callstack = Capture::GSamplingProfiler->GetCallStack(callstack_event->m_Id);
+    
+  if (!callstack) {
+    return unknown_return_text;
+  }
+
+  std::string function_name = SafeGetFormattedFunctionName(callstack->m_Data[0]);
+  std::string result = absl::StrFormat(
+    "<b>%s</b><br/><i>Sampled event</i><br/><br/><b>Callstack:</b>",
+    function_name.c_str());
+  for (auto addr : callstack->m_Data) {
+    result = result + "<br/>" + SafeGetFormattedFunctionName(addr);
```

Comment:
I suppose this can become pretty long. Is this cut off at some point? If not, should it be?

Reply:
Currently it is not cut, no. The Tooltip can get quite large and yes, it's probably a good idea to account for that, even tough I found it acceptable in my tests. 

I'd merge this one though and take a look in a new PR.